frontend/packages/dev-console/src/components/topology/components/edges/BaseEdge.tsx

```diff
@@ -17,7 +18,8 @@ type BaseEdgeProps = {
   element: Edge;
   dragging?: boolean;
   className?: string;
-} & WithRemoveConnectorProps;
+} & WithRemoveConnectorProps &
+  WithContextMenuProps;
```

Comment:
If you use `Partial` then the components that use the `BaseEdge` do not necessarily need to provide context menu props.
```suggestion
  Partial<WithContextMenuProps>;
```